### ISSUE TITLE
When using grr.dashboard.new, "name" clobbers dashboard uid

### ISSUE THREAD

**Issue by sfc-gh-shettich** [grafana/grizzly#206]
Could you leave the dashboard UID set?

to work around the 40 char uid limit I want to use:

std.substr(std.md5(user_chosen_name), 0, 40)

but still have the user chosen name show up in "grr list" so people can use "grr preview"

**Comment by malcolmholmes** [comment]
Name and UID should be the same. The right thing for Grizzly (but not yet implemented) would be either to use the name as UID (if no UID provided) or, if both name and UID provided, complain if they aren't the same.

The name is the way that we identify a resource within Grafana. But Grafana uses UIDs for lookups. Thus, they must be the same.

**Comment by sfc-gh-shettich** [comment]
Ah, I see what you mean.  Any advice for how to work with list/preview when using auto-generated UIDs?

Or, alternatively, I could pipe "show" to yq and pick out the uid and title, but the output does not follow the standard of a YAML multi-document stream (it's lacking the seperators)

**Comment by malcolmholmes** [comment]
My advice is "don't use autogenerated UIDs". I recognise that that isn't particularly helpful advice though.

Grizzly shouldn't allow dashboards without UIDs/names. If the dashboard exists already, the only (rather horrible) option is to download the dashboard, delete it, and upload it with a new UID. This is because of the fact that the "slug" must be unique as well as the UID and the slug is derived from the title. I haven't checked whether this has been fixed at all. If it has, then perhaps we can download/rename/upload/delete which is slightly less scary. I did think of implementing a "rename" feature in Grizzy to do this but got stuck on the slug issue.
